### PR TITLE
Abdelrhman: Fix mongoDB roadmap performance optimiaztion section

### PR DESCRIPTION
**Pull request by AbdelrhmanX7**
Fix
 1- when user mark Query Operators as Done it makes Performance Optimization as Done also
 2- when user click on "Performance Optimization" it opened "Query Operators" side panel (we don't have content for Performance Optimization)
 3- split Performance Optimization into another folder called 105-performance-optimization before it was under 105-query-operators

### CODE REVIEW

File: src/components/TopicDetail/TopicProgressButton.tsx [hunk 1]
```diff
@@ -149,22 +152,26 @@ export function TopicProgressButton(props: TopicProgressButtonProps) {
...
 
   if (isUpdatingProgress) {
     return (
       <button className="inline-flex cursor-default items-center rounded-md border border-gray-300 bg-white p-1 px-2 text-sm text-black">
-        <img alt="Check" className="h-4 w-4 animate-spin" src={SpinnerIcon.src} />
+        <img
+          alt="Check"
+          className="h-4 w-4 animate-spin"
+          src={SpinnerIcon.src}
+        />
```
Comment (kamranahmedse) [inline src/components/TopicDetail/TopicProgressButton.tsx:174]: Can you please revert the changes in this file

**Review by kamranahmedse — CHANGES_REQUESTED** [review]

File: src/components/TopicDetail/TopicProgressButton.tsx (hunk 1 above)
Comment (AbdelrhmanX7) [reply src/components/TopicDetail/TopicProgressButton.tsx:174]: Sure i will do that rn

Comment (AbdelrhmanX7) [reply src/components/TopicDetail/TopicProgressButton.tsx]: Done

**Comment by kamranahmedse** [comment]
Thank you!

File: src/components/TopicDetail/TopicProgressButton.tsx (hunk 1 above)
Comment (841372) [reply src/components/TopicDetail/TopicProgressButton.tsx]: > بالتأكيد سأفعل ذلك rn

**Pull request merged**